docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM openmicroscopy/ome-files-py
+
+RUN pip install datapackage
```

Comment:
Small style point: Docker creates a separate "layer" for each `COMMAND`, so for a production image it's more efficient (when downloading) to combine statements, e.g. `pip install a b c` or `pip install a && \n pip install b`, etc. For a development image this is fine though.